src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+
+    private double site;
+
+    public Square() {
+    }
+
+    public Square(String color, int site) {
```

Comment:
Consider changing the parameter type of `site` from `int` to `double` to maintain consistency with other classes and allow for non-integer side lengths.